CMakeLists.txt

```diff
@@ -98,6 +98,7 @@ if (${PROPAGATE_CONST_IS_NOT_SUBPROJECT})
             NAME test_propagate_const
             COMMAND test_propagate_const
             WORKING_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
+        include(Catch)
```

Comment:
Thanks, good catch.  I realise this doesn't show on the CI because this dependency is fetched by Conan, I'll have to think about how to not hide this error on CI going forward.